docs/lib/mosaic.md

```diff
@@ -0,0 +1,88 @@
+# Mosaic
+
+[Mosaic](https://uwdata.github.io/mosaic/) is a system for linking data visualizations, tables, and input widgets, all leveraging a database (DuckDB) for scalable processing. With Mosaic, you can interactively visualize and explore millions —and even billions— of data points.
+
+The example below shows 1 million taxi rides pickup and dropoff points in NYC on Jan 1-3, 2010. The dataset is stored in a 8MB parquet file, originally generated with a data loader.
```

Comment:
```suggestion
The example below shows 1 million taxi rides pickup and dropoff points in New York City on Jan 1–3, 2010. The dataset is stored in a 8MB [Apache Parquet](./arrow#apache-parquet) file, generated with a data loader.
```

(Dropping the “originally” for simplicity. It’s still accurate to say that it was generated by a data loader even if it’s not currently being re-generated by the data loader every time you build.)